Documentation/Migration/Index.rst

```diff
@@ -0,0 +1,29 @@
+..  include:: ../Includes.rst.txt
+
+..  _migration:
+
+=========
+Migration
+=========
+
+If you want to migrate from using Sphinx to render TYPO3 documentation to
+using the render-guides, the file :file:`Documentation/Settings.cfg` has to be
+replaced by an XML file, :file:`Documentation/guides.xml`.
+
+To facilitate migration the extension `t3docs/typo3-guides-cli` in this
+mono-repository comes with a Symfony Console Command to automatically migrate
+the outdated :file:`Settings.cfg`.
+
+With make
```

Comment:
And how do I use it without having make and ddev on my machine? We should also communicate how to do it with "raw" PHP.

Reply:
If you know please add that information. As I cannot do it I cannot add that information

Reply:
Done